docker-aoi-apache/README.md

```diff
@@ -0,0 +1,19 @@
+# AOI Apache Docker Image
+
+This utilizes version 2.4 of the Apache httpd server. The configuration file is extremely basic at present and will need to be built. As such, this has been tagged as prototype for the time being.
+
+To build locally
+
+docker build -t image-name:tag .
+
+Note: Replace image-name:tag with your desired nomenclature. Also note that you can pull the current prototype image from the docker hub with:
+
+docker pull os-wasabi/aoi-apache:prototype
```

Comment:
Is prototype still the most recent tag for this docker repo?

Reply:
If it is, I think we should consider also including a latest tag. We can version it below 1.0 if you want.

Reply:
Wrap in triple backticks